src/main/webapp/app/entities/event-crew/event-crew-detail.tsx

```diff
@@ -12,11 +12,11 @@ import { IEventCrew } from 'app/shared/model/event-crew.model';
 // tslint:disable-next-line:no-unused-variable
 import { APP_DATE_FORMAT, APP_LOCAL_DATE_FORMAT } from 'app/config/constants';
 
-export interface IEventCrewDetailProps extends StateProps, DispatchProps, RouteComponentProps<{ id: string }> {}
+export interface IEventCrewDetailProps extends StateProps, DispatchProps, RouteComponentProps<{ id: string; eventId: string }> {}
 
 export class EventCrewDetail extends React.Component<IEventCrewDetailProps> {
   componentDidMount() {
-    this.props.getEntity(this.props.match.params.id);
+    this.props.getEntity(this.props.match.params.id, this.props.match.params.eventId);
```

Comment:
sorry had to add the comment at here because cant add comment at the correct line

line 50 and line 57 button to url seems like not correct

Reply:
This file actually is not being used where there is no linkage from other pages to this page.

Reply:
but i see that in src/main/webapp/app/entities/event-crew/index.tsx you still include the class

Reply:
The link is updated and the eventCrewDetails route is commented out in commit 58422ddb864967d88daabd079ba925376621c384 and 471263d2fda83081a2d6b2c87be0402b3e77c4a2 respectively